app/pages/organization/organization-page.jsx

```diff
@@ -58,21 +60,30 @@ class OrganizationPage extends React.Component {
 
     return (
       <div className="organization-page">
+        <Helmet title={this.props.organization.display_name} />
         <section
           className="organization-hero"
           style={{ backgroundImage: `url(${this.props.organizationBackground.src})` }}
         >
-          <div className="organization-hero__container">
-            {this.props.organizationAvatar &&
-              <Thumbnail
-                src={this.props.organizationAvatar.src}
-                className="avatar organization-hero__avatar"
-                width={AVATAR_SIZE}
-                height={AVATAR_SIZE}
-              />}
-            <div>
-              <h1 className="organization-hero__title">{this.props.organization.display_name}</h1>
-              <p className="organization-hero__description">{this.props.organization.description}</p>
+          <div className="organization-hero__background-gradient">
+            {this.props.organization.announcement &&
+              <div className="informational project-announcement-banner">
+                <Markdown>
```

Comment:
The Markdown component supports a tag prop, so I think you can simplify this to this without needing a wrapping div to apply styles:
`<Markdown tag="div" className="informational project-announcement-banner">`

Reply:
oh nice! I'll make that change before merge.

Reply:
all set, thanks for heads up, good to know.